app/assets/javascripts/admin/bulk_product_update.js.coffee

```diff
@@ -39,12 +39,7 @@ angular.module("ofn.admin").controller "AdminProductEditCtrl", ($scope, $timeout
   $scope.DisplayProperties = DisplayProperties
 
   $scope.initialise = ->
-    SpreeApiAuth.authorise()
```

Comment:
how does that happen now?

Reply:
this is explained in #4111 [here](https://github.com/openfoodfoundation/openfoodnetwork/pull/4111#issuecomment-517661713)